modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java

```diff
@@ -274,19 +291,19 @@ public void execute() throws MojoExecutionException {
                 is = new FileInputStream(exclusionsFilePath);
             } else {
                 is = this.getClass().getClassLoader().getResourceAsStream(
-                        exclusionsFilePath);
```

Comment:
revert this please

Reply:
I changed to back to whatever was there in the original plugin as suggested in the review comments.

Reply:
This wouldn't be in the diff if that were true: https://github.com/gaul/modernizer-maven-plugin/blob/589ca71078fd0d90ce3f8960142c8be07f9a9c2d/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java#L277

Reply:
Somehow, I missed committing the files. Sorry for the trouble here.

Reply:
There's still added whitespace here